### DISCUSSION TITLE
Tabular + images for regression?

### DISCUSSION THREAD

**Discussion by xibux-repo** [autogluon/autogluon#1545] [Q&A] [answered]
Dear autogluon team, 

I just come across this problem, is it possible to use tabular and images for regression problem? I think the current version only support tabular+image for classification, coz in my initial try when I set the problem_type='regression', the image predictor just failed, other models are fine.  

Best
Xibin

**Comment by Innixma** [comment, marked as answer]
Thanks for creating a discussion! I have created GitHub issue #1546 to track this and ensure it is working correctly. Will mark discussion as resolved and we can continue discussion in the GitHub issue.
